lib/task_helpers/exports/tags.rb

```diff
@@ -0,0 +1,36 @@
+module TaskHelpers
+  class Exports
+    class Tags
+      # Description attribute of Tag Categories that are not visible in the UI
+      SPECIAL_TAGS = ['Parent Folder Path (VMs & Templates)', 'Parent Folder Path (Hosts & Clusters)', 'User roles'].freeze
```

Comment:
Where do these strings come from?  Could we "accidentally" change them in message catalogs or fixtures can cause you problems here?

Reply:
@jrafanie These category descriptions were determined by exporting all of the tag classifications and finding ones that are not shown in the UI (Configuration -> Settings -> ManageIQ Region -> Tags -> My Company Categories.

You are right that the descriptions could change, but when I looked at the Classification I was not able to find anything that I could use to identify them. I'm also not sure how the UI decides to not show them when displaying the tag categories. I know that they are used and maintained internally by MIQ so the goal is to prevent a user from modifying them and causing lager problems.

From what I have been able to determine the two "Parent Folder Path ..." categories are not created until an infrastructure (or maybe cloud) provider has been added and the "User roles" category is created when setting up a region.

If the description did change the only effect would be that they would be exported and could be imported.

I'm open to handling this check another way that is more robust and not dependant on the name or description of the category. I just need a little help in what to look at to make the determination.

Reply:
You should be able to do this instead -
```ruby
SPECIAL_TAGS = ["/managed/folder_path_yellow", "/managed/folder_path_blue", "/managed/user/role"]
Classification.includes(:tag).where(:parent_id => 0).where.not(:tags => {:name => SPECIAL_TAGS})
```